ORLib/ORLib/Network/ApiManager.swift

```diff
@@ -10,7 +10,6 @@ import Foundation
 public typealias ResponseBlock<T: Codable> = (_ statusCode: Int, _ object: T?, _ error: Error?) -> ()
 
 public enum ApiManagerError: Error {
-    case notFound
```

Comment:
It's been a while but I think the idea was to progressively have more specific error.
So .notFound is a more "business level" error than an HTTP status code.
I would personally keep the .notFound throwing logic as is but at the calling site explicitly handle the .notFound exception.
This will generate a bit of code duplication though.

Reply:
Understood, will revert it then